user/pages/02.deploying/01.production/docs.md

```diff
@@ -47,13 +47,13 @@ If the primary active controller goes down, a new leader will automatically be e
 Take special precautions to make sure there is always one controller running and ready, especially during host OS or orchestration platform updates and reboots.
 
 ### <a name=“persist”>Backups and Persistent Data</a>
-Be sure to periodically export the configuration file from the console and save it as a backup.
+Be sure to periodically export the configuration file from the console and save it as a backup (NOTE: As of 5.2.0, Sigstore Verifier configurations are not exportable).
```

Comment:
Clarification:
1. sigstore settings backup/restore is supported by pvc
2. sigstore settings can be exported/imported by UI: Settings -> Configuration -> Export (All) / import
3. sigstore settings export/import for CRD is not supported in 5.2.0

Reply:
@frank-at-suse Do you need to modify your edits based on above clarifications?